docs/technical_reference/unit_models/energy_models/pv_surrogate.rst

```diff
@@ -0,0 +1,34 @@
+Photovoltaic (PV) Surrogate Model
+====================================================
+
+This Photovoltaic (PV) unit model
+   * Supports steady-state only
```

Comment:
I think this requires explanation (e.g., steady state and thus assume average performance over specified time period)

Reply:
I think removing it is probably fine. I originally put that in there because I saw steady-state on some other unit models and was just trying to get some content down.